README.md

```diff
@@ -63,6 +65,17 @@ module "cloudfront" {
   origin_path                 = "/v1"
 }
 
+# Main cloudfront distribution with S3 origin with optional ACM certification and optional alias
+module "cloudfront" {
+  source                      = "voquis/cloudfront-s3-origin-acm/aws"
+  version                     = "0.0.3"
```

Comment:
Should we preempt to `0.0.4`

Reply:
We could also update the `module "acm"` from `0.0.2` to `0.0.4` :pray: https://github.com/voquis/terraform-aws-acm-dns-validation/tags

Reply:
Could we also update the `module "s3"` from `0.0.2` to `0.0.4`? :pray: 
https://github.com/voquis/terraform-aws-s3-encrypted/tags